### PR TITLE
Changed and added files according to Telegram Updated Repo

### PR DESCRIPTION
**Pull request by iamMihirT**

### CODE REVIEW

File: tests/test_scraper.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import uuid
+import telegram_scraper
```
Comment (maany) [inline tests/test_scraper.py]: import twitter scraper

File: app/scraper.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                self.logger.info(tweet.get('snippet', ''))
+                self.logger.info(tweet.get('link', ''))
+
+                filter_data = client.chat.completions.create(
+                    model="gpt-4",
+                    response_model=FilterData,
+                    messages=[
+                        {
+                            "role": "user",
+                            "content": f"Examine this tweet: {tweet['snippet']}. Is this tweet describing {filter_keyword}? "
```
Comment (maany) [inline app/scraper.py]: not defined

File: app/scraper.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
+import logging
+from logging import Logger
```
Comment (maany) [inline app/scraper.py:1]: you need to include instructor, pandas, openai, pydantic, requests in your requirements.txt

File: app/scraper.py [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+                    media_data = KernelPlancksterSourceData(
+                        name=data_name,
+                        protocol=protocol,
+                        relative_path=relative_path,
+                    )
+                    current_data = media_data
+                    output_data_list.append(media_data)
+
+                    scraped_data_repository.register_scraped_data(
```
Comment (maany) [inline app/scraper.py]: function does not exist

**Review by maany — CHANGES_REQUESTED** [review]
Please make it command line executable, fix the requirements, leave the server.py as it is ( on second thought, I will fix it for all the data pipelines later). Think of how to ensure the pipelines scrape all data, handle rate limits, etc. Use geo features if possible.

**Comment by iamMihirT** [comment]
We still have doubts regarding the usage and importing of scraping api key. Also this version does version is not yet updated with openai. We are facing trouble in replicating what is shown in the video.
But we have kept all of the code similar to telegram.

**Comment by maany** [comment]
Merging as is to unblock integration into kubeflow pipelines. Please test this and send follow up PRs ASAP.

**Comment by maany** [comment]
Make sure requirements.txt is up to date

**Pull request merged**